tests/sentry/workflow_engine/models/test_workflow.py

```diff
@@ -42,3 +43,27 @@ def test_evaluate_trigger_conditions__slow_condition(self):
 
         assert evaluation is True
         assert remaining_conditions == [slow_condition]
+
+    def test_full_clean(self):
+        self.create_workflow(
+            organization_id=self.organization.id,
+            name="test",
+            environment_id=None,
+            when_condition_group=self.create_data_condition_group(),
+            created_by_id=None,
+            owner_user_id=None,
+            owner_team=None,
+            config={"frequency": 5},
+        )
+
+        workflow2 = Workflow(
+            organization_id=self.organization.id,
+            name="test2",
+            environment_id=None,
+            when_condition_group=self.create_data_condition_group(),
+            created_by_id=None,
+            owner_user_id=None,
+            owner_team=None,
+            config={"frequency": 5},
+        )
+        workflow2.full_clean()
```

Comment:
It may be good to have a couple more tests that violate foreign key constraints or pass bad data, and validate that the `full_clean` method raises the way we expect it to.

Reply:
added a test with a bad FK key constraint (`organization` is required), the ones that were failing validation before all have `null=True` lol